docs/docs/seo-head.md

```diff
@@ -78,13 +78,13 @@ For example, if you set title in your Layout, and a title in your Page, it'll re
 
 :::info Bots & `<meta>` Tags
 
-For these headers to appear to bots and scrapers e.g. for twitter to show your title, you have to make sure your page is prerendered. If your content is static you can use Redwood's built in [Prerender](prerender.md). For dynamic tags, check the [Dynamic head tags](#dynamic-tags)
+For these headers to appear to bots and scrapers e.g. for twitter to show your title, you have to make sure your page is prerendered. If your content is static you can use Redwood's built-in [Prerender](prerender.md). For dynamic tags, check the [Dynamic head tags](#dynamic-tags)
 
 :::
 
 ## Setting `<meta>` Tags and OpenGraph Directives with `<Metadata>`
 
-Often we want to set more than just the title and description of the page—most commonly [OpenGraph](https://ogp.me/) headers.
+Often we want to set more than just the title and description of the page – most commonly [OpenGraph](https://ogp.me/) headers.
```

Comment:
👍 

Americans actually often write long hyphens (em-dash) without spaces around them. But, being European myself, I obviously prefer it *with* spaces 😉

Reply:
While reviewing this i just quickly changed it to en-dash with spaces which looked more correct in my eyes… then, on researching this in hindsight, i just found [a Merrian-Webster article](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use) insinuating this being plain wrong (though stating at the end that some printed newspapers surround an em-dash with spaces) – but then luckily finding Wikipedia stating that *en-dash with spaces* is a valid equivalent to an *em-dash w/o spaces*: https://en.wikipedia.org/wiki/Dash#Types_of_dash – so let's just argue that is the latest version of the truth. :sweat_smile: 

… typography is a science of its own.